views/components/form/select.erb

```diff
@@ -58,6 +58,10 @@ attributes ||= {} %>
     <p class="text-sm text-red-600 leading-6" id="<%= name %>-error"><%= error %></p>
   <% end %>
   <% if description %>
-    <p class="text-sm text-gray-500 leading-6" id="<%= name %>-description"><%== description %></p>
+    <% if description.start_with?("<p") %>
```

Comment:
Do we have any select descriptions starting with `<p>`?

Reply:
Yes, in `views/vm/create.erb`